test/unit/annotation_layer_spec.js

```diff
@@ -1,10 +1,47 @@
-/* globals isRef, AnnotationFactory, Dict, Name, Ref, AnnotationType,
-           AnnotationFlag, Annotation, AnnotationBorderStyle,
-           AnnotationBorderStyleType, StringStream, Lexer, Parser,
-           stringToUTF8String, AnnotationFieldFlag, PDFJS, stringToBytes */
-
+/* Copyright 2012 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 'use strict';
 
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define('pdfjs-test/unit/annotation_layer_spec', ['exports', 'pdfjs/core/primitives','pdfjs/core/annotation','pdfjs/core/stream','pdfjs/core/parser','pdfjs/shared/util','pdfjs/display/global'], factory);
+  } else if (typeof exports !== 'undefined') {
+    factory(exports, require('../core/primitives.js'),require('../core/anotation.js'),require('../core/stream.js'),require('../core/parser.js'),require('../shared/util.js'),require('../display/global.js'));
```

Comment:
The CommonJS modules are located at "../../src/core" and "../../src/display" also break down long lines at 80 chars